### DISCUSSION TITLE
Update docker image with digest only

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#26461] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

github.com

### What is your question?

I'd like to use the custom/regex manager to update a docker image which is specified by its name and digest. The tag is not used.
The following configuration doesn't work:
```
    {
      "customType": "regex",
      "fileMatch": [
        "deploy/images.csv"
      ],
      "matchStrings": [
        "CSI_SNAPSHOT_IMAGE,registry.k8s.io/sig-storage/csi-snapshotter,CSI_SNAPSHOT_VERSION,(?<currentDigest>.*)",
        "DIGEST_LIST=\"${DIGEST_LIST},registry.k8s.io/sig-storage/csi-snapshotter@sha256:(?<currentDigest>.*)\"",
        "value: registry.k8s.io/sig-storage/csi-snapshotter@sha256:(?<currentDigest>.*)",
        "- image: registry.k8s.io/sig-storage/csi-snapshotter@sha256:(?<currentDigest>.*)"
      ],
      "datasourceTemplate": "docker",
      "depNameTemplate": "registry.k8s.io/sig-storage/csi-snapshotter"
    }
```
Renovate reports:
"Regex Managers must contain currentValueTemplate configuration or regex group named currentValue"

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
Try setting currentValueTemplate to "latest"?

**Reply by lmilbaum** [reply]
Thanks @rarkins. Configuration updated as suggested.  Hitting a different error:
```
Renovate failed to look up the following dependencies: Could not determine new digest for update (docker package registry.k8s.io/sig-storage/csi-snapshotter).

Files affected: deploy/images.csv, deploy/index-image/community-kubevirt-hyperconverged/1.11.0/manifests/kubevirt-hyperconverged-operator.v1.11.0.clusterserviceversion.yaml, deploy/olm-catalog/community-kubevirt-hyperconverged/1.11.0/manifests/kubevirt-hyperconverged-operator.v1.11.0.clusterserviceversion.yaml, deploy/operator.yaml
```
https://github.com/lmilbaum/hyperconverged-cluster-operator/issues/8

**Reply by lmilbaum** [reply]
The `latest` tag is not applied to this image

**Reply by rarkins** [reply]
I can look in more detail in a couple of weeks when I'm back from vacation.

**Reply by lmilbaum** [reply]
thanks. Wishing you Marry Christmas and Happy New Year's.

**Comment by viceice** [comment]
I think you need to capture `sha256:` with the `currentDigest` too for docker datasource. 🤔
